replica/database.cc

```diff
@@ -1494,6 +1494,7 @@ database::query(schema_ptr s, const query::read_command& cmd, query::result_opti
     }
 
     column_family& cf = find_column_family(cmd.cf_id);
+    auto op = cf.read_in_progress();
```

Comment:
Is this fixing a bug?
If so, we should open an issue for it.

Reply:
Not that I'm aware of. This is more to avoid future bugs.